cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -79,8 +76,21 @@ Wallet-SDK contains implementations of these various interfaces that can be used
 fairly quickly.
 Implementations of these interfaces can be done in on the mobile side and injected in to the API methods that use them.
 A few examples of when you might want to do this:
-* Implementing platform-specific credential storage (perhaps in a secure enclave)
 * Implementing platform-specific crypto functionality (perhaps leveraging device-specific security chips)
+* Implement a custom LD document loader that uses preloaded local contexts for performance and/or security reasons.
+See [LD Document Loading](#ld-document-loading) for more information.
+
+### KeyWriter and KeyReader
+
+The KeyWriter and KeyReader interfaces appear in various places across Wallet-SDK.
+
+The `KeyWriter` interface has a single method called `create` defined on it which accepts a `keyType` argument.
+Wallet-SDK expects that `KeyWriter` implementations also store the keys they create somewhere.
+
+The `KeyReader` interface has a single method called `exportPubKey` defined on it which accepts a `keyID` argument.
+Wallet-SDK expects this method to return the public key associated with the given `keyID` argument.
+
+If either one of these behaviours is not implemented, then Wallet-SDK functionality may not work as expected.
```

Comment:
It's probably cleared to avoid using the word `may`, if possible, and be more concrete

Reply:
Done!